src/Xamarin.Android.Build.Tasks/Utilities/ManagedResourceParser.cs

```diff
@@ -299,6 +299,10 @@ void ProcessRtxtFile (string file)
 			var lines = System.IO.File.ReadLines (file);
 			foreach (var line in lines) {
 				var items = line.Split (new char [] { ' ' }, 4);
+				if (items.Length < 4) {
+					Log.LogDebugMessage ($"Ignoring '{line}' from '{file}'. It does not have the correct number of elements.");
```

Comment:
Would it be useful to know the line number?  That would be extra work to track, but it *might* be useful to know if this is the 1st line or the 1000th line…?

Reply:
Could result in a debug message of:

```
{file}:{line_number}: ignoring contents `{line}`, as it does not have the correct number of elements.
```